src/client-helpers.ts

```diff
@@ -200,7 +200,8 @@ export const cleanId = (path: string, method: string, suffix: string) => {
   return [method]
     .concat(path.split("/"))
     .concat(suffix)
-    .map((entry) => entry.replace(/[^A-Z0-9]/i, ""))
+    .map((entry) => entry.split(/[^A-Z0-9]/gi))
+    .flat()
```

Comment:
```suggestion
    .flatMap((entry) => entry.split(/[^A-Z0-9]/gi))
```

there is also `flatMap`, which is described as more efficient

> It is identical to a map() followed by a flat() of depth 1 , but slightly more efficient than calling those two methods separately.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap